### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by annalisev**
Few small changes on the main README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,12 @@
 ## Description
 > This Project is responsible for helping users to keep up with the latest game releases as well as understand the latest gaming trends.
 
-### Stakeholder Requirements
--  `Gamers` - know what the latests game releases are.
+New PC games are released every day on different platforms; it’s hard to keep up with new releases, and it’s even harder to understand the trends in gaming. We aim to create a data pipeline that tracks new new releases on major PC platforms.  
+
+
+## Stakeholder Requirements
+- `Gamers` - know what the latest game releases are.
```
Comment (hhm970) [inline README.md:9]: Good; nice, concise, and straight to the point!

**Review by hhm970 — COMMENTED** [review]
The project description could do with more detail in the other features; just a sentence or two on dashboarding and email notifications is needed in my opinion

File: README.md [hunk 2]
```diff
@@ -2,8 +2,12 @@
 ## Description
 > This Project is responsible for helping users to keep up with the latest game releases as well as understand the latest gaming trends.
 
-### Stakeholder Requirements
--  `Gamers` - know what the latests game releases are.
+New PC games are released every day on different platforms; it’s hard to keep up with new releases, and it’s even harder to understand the trends in gaming. We aim to create a data pipeline that tracks new new releases on major PC platforms.
```
Comment (annalisev) [inline README.md:5]:
```suggestion
New PC games are released every day on different platforms; it’s hard to keep up with new releases, and it’s even harder to understand the trends in gaming. We aim to create a data pipeline that tracks new new releases on major PC platforms. We will also produce a dashboard with key metrics and graphs for comparisons between different websites. Furthermore we will also send email notifications to our subscribers- weekly PDF reports and daily notifications about new releases that day.   
```

[the remark this answers is not in the document]
Comment (annalisev) [reply README.md]: Good point have done now!

File: README.md [hunk 3]
```diff
@@ -2,8 +2,12 @@
 ## Description
 > This Project is responsible for helping users to keep up with the latest game releases as well as understand the latest gaming trends.
 
-### Stakeholder Requirements
--  `Gamers` - know what the latests game releases are.
+New PC games are released every day on different platforms; it’s hard to keep up with new releases, and it’s even harder to understand the trends in gaming. We aim to create a data pipeline that tracks new new releases on major PC platforms. We will also produce a dashboard with key metrics and graphs for comparisons between different websites. Furthermore we will also send email notifications to our subscribers- weekly PDF reports and daily notifications about new releases that day.
```
Comment (hhm970) [inline README.md:5]: Good summary of our project; thanks for addressing my initial feedback!

**Review by hhm970 — APPROVED** [review]
`README.md` is descriptive and concise. I approve of this!

**Pull request merged**